src/components/ArticleCard/ArticleCard.spec.js

```diff
@@ -0,0 +1,274 @@
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+          showBookmark={false}
+        />
+      </BrowserRouter>
+    );
+
+    expect(articleCard.find('span.create-date')).toBeTruthy();
+    expect(articleCard.find('span.create-date').text()).toEqual(publishedDate);
+  });
+  it('should render an article card with an author name', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+          showBookmark={false}
+        />
+      </BrowserRouter>
+    );
+
+    expect(articleCard.find('span.read-time')).toBeTruthy();
+    expect(articleCard.find('span.read-time').text()).toEqual(readTime);
+  });
+
+  it('should not render the bookmark icon', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+          showBookmark={false}
+        />
+      </BrowserRouter>
+    );
+
+    expect(articleCard.find('button').length).toBe(0);
+  });
+
+  it('should render a bookmark icon when an article is bookmarked', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+          isBookmarked
+        />
+      </BrowserRouter>
+    );
+
+    expect(articleCard.find('button')).toBeTruthy();
+    expect(articleCard.find('button').length).toBe(1);
+    expect(articleCard.find('.bookmarked').length).toBe(1);
+  });
+
+  it('should render a bookmark icon when an article is not bookmarked', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+          isBookmarked={false}
+        />
+      </BrowserRouter>
+    );
+
+    expect(articleCard.find('button')).toBeTruthy();
+    expect(articleCard.find('button').length).toBe(1);
+    expect(articleCard.find('.not-bookmarked').length).toBe(1);
```

Comment:
If this css class doesn't actually apply styles, then you should use a data attribute on the HTML element instead e.g `data-testid`

then the test would read as:
```js
expect(articleCard.find([data-testid="not-bookmarked"]).length).toBe(1);
```